website/content/docs/nia/network-drivers.mdx

```diff
@@ -11,45 +11,141 @@ Network drivers are used by Consul-Terraform-Sync to execute and update network
 - `terraform.tfvars` - The variable definitions file is where the services input variable and any optional Consul-Terraform-Sync input variables are assigned values from Consul. It is periodically updated, typically when the task condition is met, to reflect the current state of Consul.
 - `terraform.tfvars.tmpl` - The template file is used by Consul-Terraform-Sync to template information from Consul by using the HashiCorp configuration and templating library ([hashicorp/hcat](https://github.com/hashicorp/hcat)).
 
--> **Note:** Generated template and Terraform configuration files are crucial to the automation of tasks. Any manual changes to the files may not be preserved and could be overwritten by a subsequent update.
+~> **Note:** Generated template and Terraform configuration files are crucial for the automation of tasks. Any manual changes to the files may not be preserved and could be overwritten by a subsequent update. Unexpected manual changes to the format of the files may cause automation to error.
+
+</Tab>
+<Tab heading="Terraform Cloud Driver">
+
+<EnterpriseAlert>
+  This feature requires{' '}
+  <a href="https://www.hashicorp.com/products/consul/features">Consul-Terraform-Sync Enterprise</a>{' '}
+  which is available with <strong>Consul Enterprise</strong>.
+</EnterpriseAlert>
+
+Consul-Terraform-Sync manages Terraform runs following the [API-driven run workflow](https://www.terraform.io/docs/cloud/run/api.html) for workspaces in Terraform Enterprise.
+
+On startup, Consul-Terraform-Sync:
+1. Creates or discovers Terraform Enterprise workspaces corresponding to the configured tasks.
+2. Prepares the local environment and generates Terraform configuration files that make up the root module for each task.
+3. Packages the generated files and uploads them as a configuration version for the task's workspace on Terraform Enterprise.
+
+Once all workspaces are set up, Consul-Terraform-Sync monitors the Consul catalog for service changes. When relevant changes are detected, the Terraform Cloud driver dynamically updates input variables for that task directly as [workspace variables](https://www.terraform.io/docs/cloud/workspaces/variables.html) using the Terraform Cloud API. The driver then queues a run on the workspace, with auto-apply enabled, to update your network infrastructure.
+
+~> **Note:** Although workspaces for tasks are executed in isolated environments, this does not guarantee the infrastructure changes from concurrent task executions are independent. Ensure that modules across all tasks are not modifying the same resource objects or have overlapping changes that may result in race conditions during automation.
+
+#### Remote Workspaces
```

Comment:
nothing to do with your change and no changes required - just noting an observation that links to headers on the non-default tab (Terraform Cloud Driver Tab) don't bring you to the sub-section

Reply:
🙁 i noticed that too, I'll ask around to see what can be done about it